bindings/js/Makefile

```diff
@@ -14,5 +14,7 @@ endif
 all:
 	cp $(KRML_HOME)/krmllib/js/loader.js .
 	cp ../../dist/wasm/*.{wasm,js,html} .
+	cp ../../dist/wasm/layouts.json .
```

Comment:
Why is this now needed?

Reply:
It was introduced in `dist` after the previous release